src/Rector/Contrib/PHPUnit/SpecificMethod/AssertInstanceOfComparisonRector.php

```diff
@@ -81,13 +81,13 @@ public function changeOrderArguments(MethodCall $methodCallNode): void
         /** @var Instanceof_ $comparison */
         $comparison = $oldArguments[0]->value;
 
-        $className = $comparison->class->toString();
+        $className = $comparison->class->toCodeString();
```

Comment:
This is some new method? What does it do?

Reply:
Instead of `Class`, it returns `\Class` if it so. Isn't something new I guess https://github.com/nikic/PHP-Parser/blob/master/lib/PhpParser/Node/Name.php#L99-L107

Reply:
I see, cool.

Btw, with hindsight I think we can use `toLowerString()` as well. I think the php-parse dependency would not be a problem :)

Reply:
I'll look into that